frontend/src/app/testResults/TestResultPrintModal.scss

```diff
@@ -128,7 +128,7 @@
     li {
       b {
         width: unset !important;
-        min-width: 8rem;
+        min-width: 15rem;
         max-width: 15rem;
       }
```

Comment:
If we're setting min and max to 15 - is that the equivalent to setting `width` to 15?

Reply:
yea, I just left it in case we wanna revert in a future PR, will run this by @vz3 tomorrow and will clean it up